laravel-t-shirts/app/Providers/AppServiceProvider.php

```diff
@@ -13,7 +17,16 @@ class AppServiceProvider extends ServiceProvider
      */
     public function register()
     {
-        //
+        if (env('APP_DEBUG') === false) {
```

Comment:
You need to have the check inside of the bind, you already have the function to do it.

Reply:
https://github.com/valentin7galit/my_application/pull/12/commits/57e54cb2ceb50e26686a4f7a66b90b3550b31083